digitalocean/resource_digitalocean_volume.go

```diff
@@ -36,7 +37,7 @@ func resourceDigitalOceanVolume() *schema.Resource {
 				Type:         schema.TypeString,
 				Required:     true,
 				ForceNew:     true,
-				ValidateFunc: validation.NoZeroValues,
+				ValidateFunc: validation.StringMatch(regexp.MustCompile(`^[a-z0-9]+$`), "Names must be lowercase and alphanumeric"),
```

Comment:
There might be a validator that already does this. If not works for me.

Reply:
Good point. I just checked and it looks like there isn't one.